uwblueprint/feeding-canadian-kids
# 39 Onboarding: Sign-Up Page Refactor

## Zenhub ticket link
[39 Onboarding: Sign-Up Page Refactor](https://github.com/uwblueprint/feeding-canadian-kids/issues/39)


## Implementation description
* Refactored Join.tsx where each section is contained in its own function
* Added email validation
* Added standardized placeholder text
* Fixed bug where focusing on an invalid input field would override the red error border with a blue focus border. Now, invalid input fields stay red until they are no longer in an invalid state.



## Steps to test
1. Verify that placeholder text is correct
2. Verify that validation on fields is correct (red bordered inputs for invalid fields)
3. Verify that the email validation works as expected



## What should reviewers focus on?
* Placeholder texts
* Email validation
* Code style (after refactor)


## Checklist
